xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererOMX.cpp

```diff
@@ -27,6 +27,7 @@
 #include "utils/GLUtils.h"
 #include "settings/MediaSettings.h"
 #include "windowing/WindowingFactory.h"
+#include "DVDCodecs/Video/DVDVideoCodecIMX.h"
```

Comment:
I think this is copy/paste error. This has not been tested after the refactoring.